config.w32

```diff
@@ -176,7 +191,14 @@ if (PHP_MONGODB != "no") {
     MONGOC_HAVE_RES_SEARCH: 0,
     MONGOC_CC: "",
     MONGOC_USER_SET_CFLAGS: "",
-    MONGOC_USER_SET_LDFLAGS: ""
+    MONGOC_USER_SET_LDFLAGS: "",
+    MONGOCRYPT_ENABLE_TRACE: 1,
+    MONGOCRYPT_IS_POSIX: 0,
+    MONGOCRYPT_IS_WIN: 1,
+    MONGOCRYPT_ENABLE_CRYPTO: 0,
+    MONGOCRYPT_ENABLE_CRYPTO_OPENSSL: 0,
+    MONGOCRYPT_ENABLE_CRYPTO_CNG: 0,
+    MONGOCRYPT_ENABLE_CRYPTO_COMMON_CRYPTO: 0
```

Comment:
I'd suggest prefixing this line with `// Secure Transport does not apply to Windows`, as we have earlier.